README.md

```diff
@@ -93,19 +93,50 @@ $ brew install locast2tuner
 ## Docker
 A Docker image is available from `ghcr.io/wouterdebie/locast2tuner:latest` and is built from this [Dockerfile](https://github.com/wouterdebie/locast2tuner/blob/main/assets/docker/Dockerfile).
 
-To run:
+#### To run:
 ```sh
 # Create a config directory (e.g. $HOME/.locast2tuner) and copy the example file in there:
 $ mkdir $HOME/.locast2tuner
 $ curl -o $HOME/.locast2tuner/config https://raw.githubusercontent.com/wouterdebie/locast2tuner/main/assets/config.example
-# ... edit the file ...
+# ... edit the file with your settings ...
+# Pull latest locast2tuner image:
 $ docker pull ghcr.io/wouterdebie/locast2tuner:latest
 $ docker run -p 6077:6077 -v $HOME/.locast2tuner/:/app/config --name locast2tuner -d ghcr.io/wouterdebie/locast2tuner:latest
 ```
 
-If you'd like to use `docker-compose` you can use the sample [docker-compose.yml](https://github.com/wouterdebie/locast2tuner/blob/main/assets/docker/docker-compose.yml).
+If you'd like to use `docker-compose` (_recommended_) you can use the sample [docker-compose.yml](https://github.com/wouterdebie/locast2tuner/blob/main/assets/docker/docker-compose.yml) and edit it to match your parameters.
```

Comment:
Maybe move the recommended way to the top of the section? And then add something like "If you want to do this manually.. bla bla.."?

Reply:
Or maybe even split them completely?

like:

- Docker
  - docker-compose (recommended)
    - run
    - upgrade 
  - manually
    - run
    - upgrade

Reply:
Yeah, I like your second idea - I played with some other formatting ways but was never happy.  I'll work on this in a bit.

Reply:
@wouterdebie Let me know what you think of this format.